src/js/controllers/chatController.js

```diff
@@ -61,6 +61,10 @@ module.exports = ViewController.extend({
         }
     },
 
+    updateUserIfDirty: function() {
```

Comment:
looks like it should be the default behavior of an `updateUser` function with an option to tell `force: true`